README/ReleaseNotes/v628/index.md

```diff
@@ -73,6 +73,16 @@ $ rootreadspeed --files <local-folder>/File1.root xrootd://<url-folder>/File2.ro
 
 ## Core Libraries
 
+### Support for profiling/debugging interpreted/JITted code
+
+This version of ROOT adds an LLVM JIT event listener to create perf map files
+during runtime. This allows profiling of interpreted/JITted code with cling. However,
+since this has a runtime cost, the functionality is disabled by default. The interface
+to enable it has been chosen to be via environment variables to allow both interpreted
+code in the prompt as well as code linking against ROOT to be able to optionally enable
+debugging/profiling. Using `ld.so` as example, where `LD_DEBUG` and `LD_PROFILE` exist,
+the variables `CLING_DEBUG` and `CLING_PROFILE` were chosen to enable these features.
+Setting these variables to any value enables the feature.
```

Comment:
This is a pretty heavy change, so feel free to discard or pick only parts of the suggestion:
```suggestion
This version of ROOT adds an LLVM JIT event listener to create perf map files
during runtime. This allows profiling of interpreted/JITted code generated by cling.
Instead of function addresses, the perf data will contain full function names.
In addition, frame pointers are enabled in JITted code, so full stack traces can be
generated.
Debugging is aided by switching off optimisations and adding frame pointers as well.
However, since both have a runtime cost, they are disabled by default.
Similar to `LD_DEBUG` and `LD_PROFILE` for `ld.so`, set and export the environment
variables `CLING_DEBUG` and/or `CLING_PROFILE` to enable debugging or profiling.
```

Reply:
I like the suggestion, I will use it. Thanks!